unified-codes/libs/data/src/lib/types/JWT.ts

```diff
@@ -57,19 +57,19 @@ export class JWT {
     }
   }
 
-  static parseAuthorisation(authorisation) {
-    if (!authorisation) return;
-    const [_, token] = authorisation.split(' ');
+  static parseAuthorisation(authorisation: string): JWTToken {
+    const [_, jwt] = authorisation.split(' ');
```

Comment:
which will break anonymous requests again. At the moment it's all just commented out - shall I fix up in a new issue?

Reply:
sounds good - agree, should haven't to authenticate anonymous requests